pkg/printers/internalversion/printers.go

```diff
@@ -2446,6 +2469,33 @@ func printStorageClassList(list *storage.StorageClassList, options printers.Gene
 	return rows, nil
 }
 
+func printVolumeAttributesClass(obj *storage.VolumeAttributesClass, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: obj},
+	}
+
+	name := obj.Name
+	if storageutil.IsDefaultAnnotationForVolumeAttributesClass(obj.ObjectMeta) {
+		name += " (default)"
+	}
```

Comment:
Did we consider having this information as a separate column so that the table remains easily machine readable? (Or is this an already common thing to do?)

Reply:
It follows the existing pattern, please see https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/internalversion/printers.go#L2400